indico/modules/search/result_schemas.py

```diff
@@ -68,22 +69,23 @@ def _get_url(self, data):
 
 
 class LocationResultSchema(mm.Schema):
-    venue_name = fields.String(required=True)
-    room_name = fields.String(required=True)
-    address = fields.String(required=True)
+    venue_name = fields.String(required=True)  #: The venue name
+    room_name = fields.String(required=True)  #: The room name
+    address = fields.String(required=True)  #: The address
```

Comment:
Isn't this the kind of docstring that's pretty redundant? Maybe better to just document non-obvious fields?

And I'd put them in the line before everywhere (we do the same in models where we use this type of doc string comment)

Reply:
Ideally yes. But I just can't get class properties to show up undocumented.

Reply:
`:undoc-members:` in the rst doesn't help?

Reply:
Yes, for members such as methods. Not really with class properties, unless a comment is there 😀

Reply:
sigh... maybe worth reporting as a bug in sphinx?